src/Api/Domain.php

```diff
@@ -299,4 +303,94 @@ public function verify(string $domain)
 
         return $this->hydrateResponse($response, VerifyResponse::class);
     }
+
+    /**
+     * Returns a domain tracking settings.
+     *
+     * @param string $domain name of the domain
+     *
+     * @return TrackingResponse|array|ResponseInterface
+     */
+    public function tracking(string $domain)
+    {
+        Assert::stringNotEmpty($domain);
+
+        $response = $this->httpGet(sprintf('/v3/domains/%s/tracking', $domain));
+
+        return $this->hydrateResponse($response, TrackingResponse::class);
+    }
+
+    /**
+     * Updates a domain click tracking settings.
+     *
+     * @param string $domain name of the domain
```

Comment:
Missing annotation for the 2nd argument
```suggestion
     * @param string $domain The name of the domain
     * @param string $active The status for this tracking (one of: yes, no)
```